From 93a84599a36fb10588d4c93bc4c465d179b5ee6e Mon Sep 17 00:00:00 2001
From: Antonio Ospite <ospite@studenti.unina.it>
Date: Fri, 6 Jan 2012 14:31:50 +0100
Subject: [PATCH] Don't use version number in the system_path.
X-Face: z*RaLf`X<@C75u6Ig9}{oW$H;1_\2t5)({*|jhM<pyWR#k60!#=#>/Vb;]yA5<GWI5`6u&+
 ;6b'@y|8w"wB;4/e!7wYYrcqdJFY,~%Gk_4]cq$Ei/7<j&N3ah(m`ku?pX.&+~:_/wC~dwn^)MizBG
 !pE^+iDQQ1yC6^,)YDKkxDd!T>\I~93>J<_`<4)A{':UrE

Global installation expects system patch to be something like
/usr/lib/blender/scripts/ without version numbers.
---
 source/blender/blenlib/intern/path_util.c |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/source/blender/blenlib/intern/path_util.c b/source/blender/blenlib/intern/path_util.c
index e79d850..bc6f6e6 100644
--- a/source/blender/blenlib/intern/path_util.c
+++ b/source/blender/blenlib/intern/path_util.c
@@ -89,7 +89,7 @@
 #  else /* new XDG ~/blender/.config/ */
 #    define BLENDER_USER_FORMAT			"%s/blender/%s"
 #  endif // WITH_XDG_USER_DIRS
-#  define BLENDER_SYSTEM_FORMAT			"%s/blender/%s"
+#  define BLENDER_SYSTEM_FORMAT			"%s/blender"
 #endif
 
 /* local */
@@ -1025,7 +1025,7 @@ static int get_path_system(char *targetpath, const char *folder_name, const char
 
 	system_base_path = (const char *)GHOST_getSystemDir();
 	if (system_base_path) {
-		BLI_snprintf(system_path, FILE_MAX, BLENDER_SYSTEM_FORMAT, system_base_path, blender_version_decimal(ver));
+		BLI_snprintf(system_path, FILE_MAX, BLENDER_SYSTEM_FORMAT, system_base_path);
 	}
 	
 	if(!system_path[0])
-- 
1.7.8.2

